### ISSUE TITLE
Homepage vs README

### ISSUE THREAD

**Issue by jamesmbaazam** [epiforecasts/EpiNow2#620]
Type of issue:
The README vs Website Homepage.

Detail:
In #617, we rephrased sections of the README to link out to the vignettes on the website. Those links only properly render on the home page of the website but not in the README. If this is undesirable, we can reword the README to remove the links and create [an index page](https://pkgdown.r-lib.org/articles/pkgdown.html#home-page) with the current sections in the README that only get rendered on the homepage.

In other words, do we want a different README vs Homepage?
---------

**Comment by sbfnk** [comment]
I think it's fine for these not to be rendered as links in the README.

**Comment by jamesmbaazam** [comment]
Closing as deemed unnecessary.
